package.json

```diff
@@ -77,16 +77,16 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^9.1.0",
+    "@angular/animations": "^11.2.5",
     "@angular/cdk": "^8.2.3",
-    "@angular/common": "^9.1.0",
-    "@angular/compiler": "^9.1.0",
-    "@angular/core": "^9.1.0",
+    "@angular/common": "^11.2.5",
+    "@angular/compiler": "^11.2.5",
+    "@angular/core": "^11.2.5",
```

Comment:
Remember to update the peerDepencies of `libs/designsystem/package.json` as well 🎗️